www/src/components/GitHubEditLink.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { MdModeEdit } from 'react-icons/md';
+
+const GitHubEditLink = ({
+  path = window.location.pathname.replace(/\/+$/, ''),
+}) => {
+  const url = `https://github.com/TaitoUnited/taito-cli/edit/dev/docs/${path}.md`;
+
+  return (
+    <>
```

Comment:
Maybe add some wrapper here to make sure the elements are side by side. Eg. if the outer element has `flex-direction: column;` then these would suddenly be under each other.

```js
const Wrapper = styled.div`
  display: flex;
`;
```